notifique/src/main/java/com/nathanrassi/notifique/NotifiqueApplication.java

```diff
@@ -26,4 +26,4 @@
   @Override public AndroidInjector<Service> serviceInjector() {
     return serviceInjector;
   }
-}
+}
```

Comment:
You should configure your IDE to do this. I'm surprised it is not on by default. GH will show the little red arrow when you forget to.
New lines are helpful to have at the end of files for git diffing tools.